spec/migration_spec.rb

```diff
@@ -135,7 +135,7 @@
         }
       end
 
-      it 'changes the default to Redis' do
+      it 'with changes the default to Redis' do
```

Comment:
This doesn't read well
```suggestion
      it 'changes the default to Redis' do
```